cmd/cosign/cli/verify.go

```diff
@@ -218,43 +218,43 @@ You may specify either a key, a certificate or a kms reference to verify against
 
 The signature may be specified as a path to a file or a base64 encoded string.
 The blob may be specified as a path to a file or - for stdin.`,
-		Example: `  cosign verify-blob (--key <key path>|<key url>|<kms uri>)|(--cert <cert>) --signature <sig> <blob>
+		Example: ` COSIGN_EXPERIMENTAL=1 cosign verify-blob (--key <key path>|<key url>|<kms uri>)|(--cert <cert>) --signature <sig> <blob>
 
   # Verify a simple blob and message
-  cosign verify-blob --key cosign.pub --signature sig msg
+  COSIGN_EXPERIMENTAL=1 cosign verify-blob --key cosign.pub --signature sig msg
 
   # Verify a simple blob with remote signature URL, both http and https schemes are supported
-  cosign verify-blob --key cosign.pub --signature http://host/my.sig
+  COSIGN_EXPERIMENTAL=1 cosign verify-blob --key cosign.pub --signature http://host/my.sig
 
   # Verify a signature from an environment variable
-  cosign verify-blob --key cosign.pub --signature $sig msg
+  COSIGN_EXPERIMENTAL=1 cosign verify-blob --key cosign.pub --signature $sig msg
 
   # verify a signature with public key provided by URL
-  cosign verify-blob --key https://host.for/<FILE> --signature $sig msg
+  COSIGN_EXPERIMENTAL=1 cosign verify-blob --key https://host.for/<FILE> --signature $sig msg
 
   # Verify a signature against a payload from another process using process redirection
-  cosign verify-blob --key cosign.pub --signature $sig <(git rev-parse HEAD)
+  COSIGN_EXPERIMENTAL=1 cosign verify-blob --key cosign.pub --signature $sig <(git rev-parse HEAD)
 
   # Verify a signature against Azure Key Vault
```

Comment:
I would like to combine all the KMS examples. It'd be nice to list the syntax for each, but I don't need it in the full context.